### ISSUE TITLE
line_terminator not lineterminator use in .to_csv from pandas throws error with pandas > 1.5

### ISSUE THREAD

**Issue by hyphaltip** [KwanLab/Autometa#350]
autometa-binning-summary throw error because of different expectation for .to_csv - I assume this is a pandas version problem I have? per https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html which changed after 1.5.0 but the env requirements state only pandas > 1.1 (https://github.com/KwanLab/Autometa/blob/main/autometa-env.yml) 

```
+ autometa-binning-summary --binning-main results_autometa_mh/Boulby_Sample_1/autometa/Boulby_Sample_1.gtdb.archaea.dbscan.main.tsv --markers results_autometa_mh/Boulby_Sample_1/autometa/Boulby_Sample_1.archaea.markers.tsv --metagenome results_megahit/Boulby_Sample_1/final.contigs.fa --dbdir /srv/projects/db/autometa
/2.0/gtdb/ --dbtype gtdb --output-stats results_autometa_mh/Boulby_Sample_1/autometa/Boulby_Sample_1.gtdb_archaea_metabin_stats.tsv --output-taxonomy results_autometa_mh/Boulby_Sample_1/autometa/Boulby_Sample_1.gtdb_archaea_metabin_taxonomy.tsv --output-metabins results_autometa_mh/Boulby_Sample_1/autometa/Boulby_Sam
ple_1.gtdb_archaea_metabins
[11/12/2023 02:18:59 PM INFO] autometa.binning.summary: Retrieving metabins' stats for cluster
[11/12/2023 02:18:59 PM INFO] root: Wrote metabin stats to results_autometa_mh/Boulby_Sample_1/autometa/Boulby_Sample_1.gtdb_archaea_metabin_stats.tsv
[11/12/2023 02:19:01 PM DEBUG] autometa.taxonomy.gtdb: Processing nodes from /srv/projects/db/autometa/2.0/gtdb/merged.dmp
[11/12/2023 02:19:01 PM DEBUG] autometa.taxonomy.gtdb: merged loaded
[11/12/2023 02:19:01 PM DEBUG] autometa.taxonomy.gtdb: Processing delnodes from /srv/projects/db/autometa/2.0/gtdb/delnodes.dmp
[11/12/2023 02:19:01 PM DEBUG] autometa.taxonomy.gtdb: delnodes loaded
[11/12/2023 02:19:01 PM INFO] autometa.binning.summary: Retrieving metabin taxonomies for cluster
Traceback (most recent call last):
  File "/opt/linux/rocky/8.x/x86_64/pkgs/autometa/2.2.1/env/bin/autometa-binning-summary", line 10, in <module>
    sys.exit(main())
  File "/bigdata/operations/pkgadmin/opt/linux/centos/8.x/x86_64/pkgs/autometa/2.2.1/env/lib/python3.10/site-packages/autometa/binning/summary.py", line 402, in main
    taxa_df = get_metabin_taxonomies(
  File "/bigdata/operations/pkgadmin/opt/linux/centos/8.x/x86_64/pkgs/autometa/2.2.1/env/lib/python3.10/site-packages/autometa/binning/summary.py", line 258, in get_metabin_taxonomies
    bin_df[outcols]
TypeError: NDFrame.to_csv() got an unexpected keyword argument 'line_terminator'

===
Conda version installed is `pandas                    2.1.2`

**Comment by WiscEvan** [comment]
Hi Jason, sorry for the late response. I have not investigated this deeply, but it looks like you should be able to pin pandas to stay within version 1.1 and 1.5

You should be able update your environment directly using `mamba` or you can edit the `autometa-env.yml` as you mentioned.

```python
# in autometa-env.yml
pandas >1.1,<1.5
```

I think this should resolve your issue. Hopefully a PR will be incoming soon. Thanks for bringing this to our attention.

**Comment by Sidduppal** [comment]
Issue should be fixed by PR #353
